### DISCUSSION TITLE
How to access Organisations (and Passwords) as Admin?

### DISCUSSION THREAD

**Discussion by Seb-AM** [dani-garcia/vaultwarden#2013] [Q&A] [answered]
Hypothetical case:
So lets say that a Team Member leaves the company and has as inept as he was created an Organisation only he could access.
So I am Admin of the Vaultwarden instance, i can see his Organisation just fine. How can I assign a new organisation owner, or can extract the passwords? Dump the database in clear text?
Any more elegant way?

To be clear this case is really hypothetical, but my company is interested to use vaultwarden and we want to know how to handle that case.

**Comment by BlackDex** [comment, marked as answer]
You can only change the status of someone if that person is already a member of the organization, else there is no way to access that organization. There is no magical way to invite your self into that organization since keys are generated by the web-vault interface.

Dumping the database in clear-text would be bad for business for a password manager, so that is really not possible.

**Reply by Seb-AM** [reply]
Many thanks for the quick answer.

**Comment by cksapp** [comment]
As an aside to prevent possible future issues of this I would recommend adding the environment variable "**- ORG_CREATION_USERS=none**"
This can also be set within the Vaultwarden admin page, to allow only certain user emails, for admins to create Orgs but prevent others from creating organizations in the future that have not been set up or controlled by the admin.
